src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()">
+      <fieldset class="set-field">
+        <legend>Personal information</legend>
+        <label
+          for="Surname"
+          class="form-field2"
+          tabindex="1">
+
+          Surname:
+          <input
+            id="Surname"
+            class="field"
+            type="text"
+            name="Surname"
+            required>
+        </label>
```

Comment:
Follow the code style guidelines
https://mateacademy.slack.com/archives/C02CSGP1KD1/p1630943541196100?thread_ts=1630943169.192800&cid=C02CSGP1KD1
Also, don't start id and name with capital letter.

```suggestion
        <label
          for="Surname"
          class="form-field2"
          tabindex="1"
        >
          Surname:
          <input
            id="surname"
            class="field"
            type="text"
            name="surname"
            required
          >
        </label>
```